dashboard/test/models/census/census_summary_test.rb

```diff
@@ -262,21 +262,38 @@ class Census::CensusSummaryTest < ActiveSupport::TestCase
     validate_summary(school, school_year, "NO")
   end
 
-  test "High school lack of state data in a denylist state does not override anything" do
+  test "High school lack of state data in a denylist state and year does not override anything" do
     return if Census::StateCsOffering::INFERRED_NO_EXCLUSION_LIST.empty?
-    school_year = 2020
+    school_year = 2021
+    state = Census::StateCsOffering::INFERRED_NO_EXCLUSION_LIST.first.split(':').first
     school = create :census_school,
       :with_teaches_no_teacher_census_submission,
       :with_teaches_yes_teacher_census_submission,
       :with_teaches_yes_parent_census_submission,
       :with_one_year_ago_teaches_yes,
       :with_two_years_ago_teaches_yes,
       :with_three_years_ago_teaches_yes,
-      state: Census::StateCsOffering::INFERRED_NO_EXCLUSION_LIST.first,
+      state: state,
       school_year: school_year
     validate_summary(school, school_year, "MAYBE")
   end
 
+  test "High school lack of state data in a denylist state and non-denylist year does not override anything" do
```

Comment:
Know you're in a hurry so can leave this comment for someone else to deal with, but I think this test is showing that we **do override** when we're in a state-year that is not in the deny list?